packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md

```diff
@@ -105,17 +105,17 @@ Example:
 <img alt="{{orgName}}" src="https://example.com//SomeOtherImage.png" class="org-logo">
 ```
 
-### <span v-pre>`{{{errorSummary}}}`</span>
+### <span v-pre>`{{errorSummary}}`</span>
 
-### <span v-pre>`{{{errorDescription}}}`</span>
+### <span v-pre>`{{errorDescription}}`</span>
```

Comment:
it should be `{{{` as we have this in our html template: https://github.com/okta/okta-core/blob/master/resources/src/main/resources/templates/custom_pages/error.template.html#L36. only `errorDescription` have 3 brackets. `errorSummary` change looks good

`{{{`: All variables are HTML escaped by default. If you want to return unescaped HTML, use the triple mustache: {{{name}}}.
http://mustache.github.io/mustache.5.html